PasswordValidator/Controllers/PasswordValidationController.cs

```diff
@@ -8,7 +8,8 @@ public class PasswordValidationController : Controller
         [Route("ValidatePassword")]
         public ActionResult ValidatePassword(string password)
         {
-            bool isValid = Validator.IsValidPassword(password);
+            SimpleValidator simpleValidator = new SimpleValidator();
```

Comment:
You declare the variable of type interface not the concrete class

`IPasswordValidator validator = new SimpleValidator();`